Utilities/Install-CustomScript.ps1

```diff
@@ -0,0 +1,177 @@
+##############################################################################################
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# Install-CustomScript.ps1
+<#
+.SYNOPSIS
+
+.PARAMETER
+	-AzureSecretsFile, the path of Azure secrets file
```

Comment:
@lizzha how can you test in all possible scenario? Also add description in PR.

Reply:
I tested for the below scenarios, for both Linux and Windows VMs
1. single VM in a resource group
2. all VMs in a resource group
3. all VMs in a subscription